src/ast.rs

```diff
@@ -99,6 +136,7 @@ impl Default for CompilationUnit {
 #[derive(Debug, Copy, PartialEq, Clone)]
 pub enum VariableBlockType {
     Local,
+    Temp,
```

Comment:
We can, in methods it is equivalent to VAR blocks

Reply:
They are similar to VAR in functions not in POUs since functions get allocated on every execution. We might want to reconsider them later, you can treat them as VAR for now, but this will leak into other POUs which is why I hoped we could tackle this in its own story.
Questions I have there is how do temps in POUs work? Do TEMP work in classes?